src/popup-page/modules/app.tsx

```diff
@@ -1,10 +1,15 @@
 import BeeDashboard from '@ethersphere/bee-dashboard'
 import React, { useContext, useEffect, useState } from 'react'
+import { ExternalLink } from 'react-feather'
+
 import { getItem } from '../../utils/storage'
 import { GlobalContext } from '../context/global'
-import { BeeApiUrlChangeForm } from './bee-api-url-change-form'
-import { PostageBatchElement } from './postage-batch-element'
-import { Web2Origin } from './web2-origin'
+import GlobalPostageStamp from './GlobalPostageStamp'
+import Web2Origin from './Web2Origin'
+import Logo from '../assets/logo.svg'
```

Comment:
adjust ts configuration for this one, because it shows error now:
`Cannot find module '../assets/logo.svg' or its corresponding type declarations.ts(2307)`

Reply:
This would require alterations of `tsconfig` which causes a lot of other issues. So I decided to instead just put the SVG directly into a react component.